csun-metalab/innovation
# IWE-44 - Fix for Mailers in Latest Laravel Version

### `IWE-44` - `PR`
`**READY**`

### Migrations
`NO`

### What's this PR do?
`Adapted old Mailer class to utilize new new method of sending emails using a Generic Mail Class.`
### Where should the reviewer start?  
`You will have to test fire emails for invitations, feedback , etc. `

### Related PRs
`NONE`

### Requirements & Dependencies

`NONE`

### Steps to Test or Reproduce
`Manually visit pages to ensure emails being fired.`
**YOU MAY change the .env value APP_FEEDBACK_TO= , into your email address.**

### Impacted Areas in Application
`Email Functionality.`
